update_ta_aliases_list.py

```diff
@@ -0,0 +1,53 @@
+# import json  # for testing purposes
+import argparse
+from ta_list_methods import get_all_ta_from_act, get_all_alias_facts_from_act, \
+    add_ta_to_map, add_ta_alias_to_map, create_config
+
+
+def parseargs():
+    """ Parse arguments """
+    parser = argparse.ArgumentParser(description='Updates list of threat actor \
+        aliases from an ACT platform')
+    parser.add_argument('--baseurl', required=True, help='URL to ACT platform \
+        installation')
+    parser.add_argument('--userid', required=True, help="User ID for ACT \
+        platform")
+
+    return parser.parse_args()
+
+
+def main():
+
+    args = parseargs()
+
+    # gets all ta names from objects(as a set of strings) and facts(as a
+    # set with tuples of two strings) in ACT.
+
+    threatactors = get_all_ta_from_act(args.baseurl, args.userid)
+
+    # gets all threat actor aliases from act platform
+
+    ta_aliases = get_all_alias_facts_from_act(args.baseurl, args.userid)
+
+    # save ta and ta_aliases to json test file
```

Comment:
Could these statements be bound to an extra command line argument, e.g. "output_json", so that if you execute the script with `--output-json` and test for "args.output_json" here the statements would be executed?